nomad/state/events_test.go

```diff
@@ -39,6 +39,56 @@ func TestEventFromChange_SingleEventPerTable(t *testing.T) {
 	out := eventsFromChanges(s.db.ReadTxn(), changes)
 	require.Len(t, out.Events, 1)
 	require.Equal(t, out.Events[0].Type, structs.TypeJobRegistered)
+}
+
+// TestEventFromChange_NodeSecretID ensures that a node's secret ID is not
+// included in a node event
+func TestEventFromChange_NodeSecretID(t *testing.T) {
+	t.Parallel()
+	s := TestStateStoreCfg(t, TestStateStorePublisher(t))
+	defer s.StopEventBroker()
+
+	node := mock.Node()
```

Comment:
I feel like there should be an assertion here that the SecretID is not empty before asserting that it is, unless you think that's unnecessary.

Reply:
:+1: added assertion